doc/source/tutorials/designpatterns.rst

```diff
@@ -57,7 +57,7 @@ Here, we introduce IF/ELSE logic into UNTIL loops:
     STAGE.
     UNTIL SHIP:ALTITUDE > 20000 {
         IF SHIP:ALTITUDE > 10000 {
-            LOCK STEERING TO R(0,0,-90) + HEADING(90,45).
+            LOCK STEERING TO TO R(0,0,-90) + HEADING(90,45).
```

Comment:
You can fix the obvious error here if you like so you can merge it.  I wont be at my computer until tomorrow.